src/webots/nodes/WbEmitter.cpp

```diff
@@ -105,7 +107,31 @@ void WbEmitter::updateRange() {
   mNeedToSetRange = true;
 }
 
+bool WbEmitter::is_channel_allowed() {
```

Comment:
Method names should written in [lower camel case](https://github.com/cyberbotics/webots/wiki/CPP-Coding-Style#cs306-names-representing-methods-or-functions-must-be-written-in-lower-camel-case)
```suggestion
bool WbEmitter::isChannelAllowed() {
```